examples/calendar/backend/models/appointment.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Model } from "sequelize";
+
+export class Appointment extends Model {
+	public id!: number;
```

Comment:
do we need explicit `public` access modifier?

Reply:
Helps with future work in case more complexity needs to be added, good practise by default.  I used it mainly because it was in the boilerplate code.

Reply:
Although we don't need this, but not a blocker at the moment